src/app/layout.tsx

```
@@ -26,13 +27,16 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body
-        className={`${geistSans.variable} ${geistMono.variable} antialiased`}
-      >
-        <RouteChangeLoader />
-        {children}
-        <BackTop />
-      </body>
+      <ImageKitProvider
+        urlEndpoint={process.env.NEXT_PUBLIC_IMAGEKIT_URL_ENDPOINT || ""}>
```

It should throw an error if `NEXT_PUBLIC_IMAGEKIT_URL_ENDPOINT` does not exist.

Doing it this way will avoid the error, but can cause unexpected results since it will not work without us knowing the direct root cause.